libs/novui/src/recipes/modal.recipe.ts

```diff
@@ -0,0 +1,50 @@
+import { defineSlotRecipe } from '@pandacss/dev';
+import { ModalBaseStylesNames } from '@mantine/core';
+
+const SLOTS: ModalBaseStylesNames[] = ['body', 'header', 'title', 'overlay', 'root', 'content', 'close', 'inner'];
+
+export const MODAL_RECIPE = defineSlotRecipe({
+  className: 'modal',
+  jsx: ['Modal'],
+  slots: SLOTS,
+  base: {
+    root: {
+      colorPalette: 'mode.cloud',
+      borderRadius: '75',
+      position: 'relative',
+      padding: '50',
+      color: 'typography.text.main',
+      width: 'modal.width !important',
+    },
+    inner: {
+      width: '100%',
+    },
+    header: {
+      background: 'modal.background !important',
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice job with using a new token here!‏